app/main/posts/collections/collections-routes.js

```diff
@@ -0,0 +1,98 @@
+module.exports = [
+    '$stateProvider',
+    function (
+        $stateProvider
+    ) {
+
+    $stateProvider
+    .state(
+        {
+            url: '^/collections/:id',
```

Comment:
this is just a state that takes you to the right view if you don't tell it where you need to go.